cmd/tfplugindocs/main_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package main
+
+import (
+	"os"
+	"testing"
+
+	"github.com/rogpeppe/go-internal/testscript"
+
+	"github.com/hashicorp/terraform-plugin-docs/internal/cmd"
+)
+
+func TestMain(m *testing.M) {
+	os.Exit(testscript.RunMain(m, map[string]func() int{
+		"tfplugindocs": cmd.Main,
+	}))
+}
+
+func Test_GenerateAcceptanceTests(t *testing.T) {
+	t.Parallel()
+	if os.Getenv("ACCTEST") == "" {
+		t.Skip("ACCTEST env var not set; skipping acceptance tests.")
+	}
```

Comment:
The tests don't take too long to run right now although I can see a future where that could be the case. The flag is mainly there because of the custom temporary directory without the `os.TempDir()`. I wanted the test to be opt-in, with the understanding of where the temporary directory would be created, but it's not ideal.